src/_assets/css/_content.scss

```diff
@@ -1,3 +1,10 @@
+@mixin header-clearance {
```

Comment:
I wonder if this mixin should be at a higher level file, because we may use this in the homepage or showcases page as well, which aren't content pages.

Reply:
I suppose that we can move it when the need arises (though `_landing-page` comes after `_content` in `main.scss` so that would still work.